musicdroid/src/main/java/org/catrobat/musicdroid/pocketmusic/SplashScreen/SplashScreen.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Musicdroid: An on-device music generator for Android
+ * Copyright (C) 2010-2015 The Catrobat Team
+ * (<http://developer.catrobat.org/credits>)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * An additional term exception under section 7 of the GNU Affero
+ * General Public License, version 3, is available at
+ * http://developer.catrobat.org/license_additional_term
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.catrobat.musicdroid.pocketmusic.SplashScreen;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.os.Bundle;
+import android.os.Handler;
+
+
+import org.catrobat.musicdroid.pocketmusic.R;
+import org.catrobat.musicdroid.pocketmusic.instrument.piano.PianoActivity;
+
+public class SplashScreen extends Activity {
+
+    private static int SPLASH_TIME_OUT = 1500;
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.splash);
+        new Handler().postDelayed(new Runnable() {
+            @Override
+            public void run() {
+                Intent launchpianoactivity = new Intent(SplashScreen.this, PianoActivity.class);
```

Comment:
Variables should be written in CamelCase that means launchPianoActivity in this case